src/main/java/org/ibp/api/java/impl/middleware/inventory/common/validator/InventoryCommonValidator.java

```diff
@@ -82,4 +83,17 @@ public void validateLotNotes(final String notes, final BindingResult errors) {
 		}
 	}
 
+	public void validateLotNotes(final List<String> notes, final BindingResult errors) {
+		if (notes.stream().anyMatch(s -> StringUtils.isBlank(s))) {
+			errors.reject("lot.input.list.notes.null.or.empty", "");
+			throw new ApiRequestValidationException(errors.getAllErrors());
+		}
+
+		final List<String> filteredNotes =
```

Comment:
This validation will be better implemented with findAny, we dont need in this case to know the number of invalid notes.